blog/steering/[BLOG]-toc-2021-election-announcement.md

```diff
@@ -0,0 +1,42 @@
+It’s time to kick off the Knative Technical Oversight Committee (TOC) elections
+again for 2021, and we are ready to accept nominations!
+
+This election will shape the future of Knative as a community and project. While
+WGs help shape the individual direction of the project, the [TOC covers the
```

Comment:
the link seems quite long, just curious if it would be easier on the eyes with:
the [TOC](...TECH-OVER...) covers the technical direction, health of the project...
so just have the TOC be the link portion.

Reply:
Good point. I'll fix that.